pkg/monitoring/rules/alerts/vms.go

```diff
@@ -128,5 +128,30 @@ var (
 				operatorHealthImpactLabelKey: "none",
 			},
 		},
+		{
+			Alert: "GuestFilesystemAlmostOutOfSpace",
+			Expr:  intstr.FromString("kubevirt_vmi_filesystem_used_bytes / kubevirt_vmi_filesystem_capacity_bytes >= 0.85 < 0.95"),
```

Comment:
@machadovilaca  If you change this to"(kubevirt_vmi_filesystem_used_bytes / kubevirt_vmi_filesystem_capacity_bytes)*100 >= 85 < 95"
You would be able to add the current value to the alert description with
(current: {{ $value }}%)